Kernel/Arch/i386/CPU.cpp

```diff
@@ -920,15 +920,13 @@ UNMAP_AFTER_INIT void write_xcr0(u64 value)
 
 READONLY_AFTER_INIT FPUState Processor::s_clean_fpu_state;
 
```

Comment:
Typo in commit message, Their's -> There's.

Reply:
Yup, fixed, thanks!